Source/WebCore/platform/graphics/holepunch/MediaPlayerPrivateHolePunch.h

```diff
@@ -93,20 +93,24 @@ class MediaPlayerPrivateHolePunch : public MediaPlayerPrivateInterface, public C
 
     void pushNextHolePunchBuffer();
     void swapBuffersIfNeeded() final;
+    void setNetworkState(MediaPlayer::NetworkState);
 #if !USE(NICOSIA)
     RefPtr<TextureMapperPlatformLayerProxy> proxy() const final;
 #endif
 
+protected:
+    MediaPlayer* m_player;
```

Comment:
Why is this change to "protected" needed? Is there going to be a MediaPlayerPrivateHolePunch subclass somewhere?

Reply:
This was a straightforward cherry-pick. I don't remember why it was needed in 2.22 branch, but it looks like I can remove it in 2.28. I'll update the commit,